### PR TITLE
Added 2 test additionally to auth-flow.spec.ts

### PR DESCRIPTION
**Pull request by mon-bene**

### CODE REVIEW

**Review by alekseigolov1 — APPROVED** [review]
Excellent job :)  update readme file with new tests added as well . Tests itself looks good to me

**Comment by vladimir-tl** [comment]
pls merge once get approval !

**Comment by Marusia2** [comment]
@mon-bene Great job :) Approve your fixes, like the way your checklist looks like:)

**Review by Marusia2 — APPROVED** [review]

**Pull request merged**
